src/nrniv/nvector_nrnparallel_ld.h

```diff
@@ -65,13 +65,7 @@ mv temp nvector_nrnparallel_ld.cpp
 
 
 #include <nrnmpiuse.h>
-#if NRNMPI_DYNAMICLOAD
-#define MPI_DOUBLE double
-#define MPI_LONG   long
-#define MPI_Comm   int
-#else
-#include <mpi.h>
-#endif
+#define MPI_Comm int
```

Comment:
I'm not sure this is relevant, but it is not always the case that MPI_Comm is an int (e.g. see openmpi). This is the point where my efforts to update to more recent version of sundials (version 6) have foundered.

Reply:
My future plan is to remove MPI_Comm from nrn side, should be only on nrnmpi side.